src/main/kotlin/app/revanced/patches/idaustria/patch/RootDetectionPatch.kt

```diff
@@ -0,0 +1,22 @@
+package app.revanced.patches.idaustria.patch
```

Comment:
Please move this patch and other related code to a proper package such as `app.revanced.patches.idaustria.detection.root`